src/ConvertESOMTR/mainModule.f90

```diff
@@ -430,6 +430,7 @@ Subroutine readESOdictionary()
 !'check to make sure the entire file wasn't treated as the data dictionary
 If (lineOfESO /= 'End of Data Dictionary') Then
```

Comment:
Interesting, I clicked the Add Single Comment button and my screen greyed out, then I clicked it again. Then I saw what you described and clicked Cancel thinking I did not post a comment. In any event I agree with you this section of code only reads the data dictionary.